controller/Lists.php

```diff
@@ -319,10 +323,13 @@ public function saveLists(ValueCollectionService $valueCollectionService): void
             )
         );
 
-        $listElements = array_filter($payload, function (string $key) {
-            return (bool)preg_match('/^list-element_/', $key);
-        },
-        ARRAY_FILTER_USE_KEY);
+        $listElements = array_filter(
```

Comment:
is it possible to increase the coverage of automated tests of this class?

Reply:
I do not think it is feasible to cover all the controller by test in scope of this ticket. Also I have seen a task for redesign the lists index page https://oat-sa.atlassian.net/browse/ADF-790 in an epic https://oat-sa.atlassian.net/browse/ADF-792 I suppose those are better candidates to include improving the test coverage for that area into their scope.